src/lib/components/forms/BulkActions.svelte

```diff
@@ -79,7 +82,7 @@ Most actual actions are deferred to their own forms, so this is more of a switch
     {#each Object.entries(actions) as [action, label]}
       <SidebarItem
         hover
-        disabled={$selected?.length < 1}
+        disabled={!$me || $selected?.length < 1}
```

Comment:
Preventing the modal from showing if `!$me` will help prevent users from undoing the disabled attr. Since the forms won't be in the DOM, they can't reveal them and submit them.